pkg/apis/build/v1alpha1/build_defaults_test.go

```diff
@@ -0,0 +1,16 @@
+package v1alpha1
+
+import (
+	"testing"
+)
+
+func TestSetDefault(t *testing.T) {
```

Comment:
Can you also test that a build that specifies both values doesn't have its values overwritten?